sts/internal/types.go

```diff
@@ -365,31 +396,54 @@ func (c *Conditions) C14N() string {
 	if c.ProxyRestriction != nil {
 		names = append(names, &c.ProxyRestriction.XMLName)
 	}
-	if c.RenewRestrictionType != nil {
-		names = append(names, &c.RenewRestrictionType.XMLName)
+
+	for i := range c.Condition {
+		switch r := c.Condition[i].(type) {
```

Comment:
Hi @dougm,

I haven't looked, but I assume there are the only two `Condition` types? No default case is needed?